exoscale/resource_exoscale_compute.go

```diff
@@ -206,12 +206,14 @@ func resourceComputeCreate(d *schema.ResourceData, meta interface{}) error {
 
 	client := GetComputeClient(meta)
 
-	displayName := d.Get("display_name").(string)
+	name := d.Get("name").(string)
 	hostName := regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9\-]+$`)
```

Comment:
I mean this constraint should not apply to display name I guess.

Reply:
Ah, right: indeed it'll only be applied for the `hostname` attribute.